src/components/award-icon-library.js

```diff
@@ -350,6 +355,7 @@ class AwardIconLibrary extends BaseMixin(LitElement) {
 				slot="footer"
 				@click=${this._createIcon}
 				primary
+				.disabled = ${!(this.isValidImage && this.isValidIconName)}
```

Comment:
nit: I'm not sure if `=` works with spaces around it in all browsers. It's also just a slight code style inconsistency.  
```suggestion
				.disabled=${!(this.isValidImage && this.isValidIconName)}
```